### ISSUE TITLE
CallToAction icon misplaced on pfsense login page

### ISSUE THREAD

**Issue by garrettboone** [passbolt/passbolt_browser_extension#136]
## CallToAction icon misplaced on pfsense login page
* Passbolt Version: 3.4/3.4
* Platform and Target:
-- Browser: FFDev96
-- Web server: NGINX

### What you did
Login for pfsense is still showing misplaced icon:
![image](https://user-images.githubusercontent.com/47018474/145228221-3679da77-a2d2-494f-81af-06b136534a42.png)

### What happened
Here is html for the page:
<details>
  <summary>Here is html for the login page:</summary>
  
  ```
  <body id="login" >
		<div id="total">
			<header>
				<div id="headerrow">
					<div class="row">
						<!-- Header left logo box -->
						<div class="col-sm-4">
							<div id="logodiv" style="text-align:center" class="nowarning">
								<svg id="logo" role="img" aria-labelledby="pfsense-logo" x="0px" y="0px" viewBox="0 0 282.8 84.2">
	<title id="pfsense-logo-svg">pfSense Logo</title>

							</div>
						</div>

						<!-- Header center message box -->
						<div class="col-sm-4 nowarning msgbox text-center text-danger">
						</div>

						<!-- Header right message box (hostname or msg)-->
						<div class="col-sm-4 nowarning msgbox text-center">
							<span id="hostspan">
								<a><h4>Login to pfSense</h4></a>
							</span>
						</div>
					</div>
	            </div>
	        </header>

	        <div style="background: #333333;" class="pagebody">
	        	<div class="col-sm-4"></div>

	        	<div class="col-sm-4 offset-md-4 logoCol">
					<div class="loginCont center-block">
		                <form method="post" autocomplete="off" class="login"><input type='hidden' name='__csrf_magic' value="sid:3f073998312121c508566cdc3304620b40c40618,1638973436" />
			                <p class="form-title">Sign In</p>
			                <input name="usernamefld" id="usernamefld" type="text" placeholder="Username" autocorrect="off" autocapitalize="none"/>
			                <input name="passwordfld" id="passwordfld" type="password" placeholder="Password" />
			                <input type="submit" name="login" value="Sign In" class="btn btn-success btn-sm" />
		                </form>
					</div>
	            </div>

	        	<div class="col-sm-4"></div>
	        </div>

	        <footer id="3">
	            <div id="footertext">
					<p class="text-muted">
						<a target="_blank" href="https://pfsense.org">pfSense</a> is developed and maintained by <a target="_blank" href="https://netgate.com">Netgate. </a> &copy; ESF 2004 - 2021<a target="_blank" href="https://pfsense.org/license"> View license.</a>					</p>
	            </div>
	        </footer>
	    </div>
  </body>
  ```
</details>

<details>
  <summary>Here is css for the page:</summary>
  
  ```
@import url("/css/logo.css");

body, html {
    height: 100%;
    padding: 0;
    margin: 0;
}

body {
    width: 100%;
}

header {
}

#headerrow {
    position: fixed;
    height: 90px;
    top: 0;
    width: 100%;
    background-color: white;
}

.pagebody {
    position: absolute;
    top:90px;
    bottom:25px;
    width: 100%;
    color: white;
}

.pagebodywarn {
    position: absolute;
    top:140px;
    bottom:25px;
    width: 100%;
    color: white;
}

.nowarning {
	height: 80px;
	padding-top: 10px;
}

#hostspan {
	text-align: right;
}

.msgbox {
	padding-right: 60px;
	padding-top: 25px;
}

@media only screen and (max-width : 768px) {
    /* only size 'xs' and below */
    #headerrow {
        height: 100px;
    }

    .pagebody {
    	top: 100px;
    }

    .pagebody2 {
		top: 250px;
    }

    .nowarning {
		height: 60px;
	}

	.msgbox {
		padding-right: 0px;
		padding-top: 0px;
	}

	#hostspan {
		text-align: center;
	}
}

#footertext {
    position: fixed;
    height: 25px;
    bottom: 0;
    width: 100%;
    background-color: #212121;
    color: white;
    text-align: center;
}

.loginCont {
    position: absolute;
    top: 50%; left: 50%;
    transform: translate(-50%,-50%);
    height: 55%;
    width: 80%;
}

.error-panel a {
	color: white;
}
p.form-title
{
    font-family: 'Open Sans' , sans-serif;
    font-size: 20px;
    font-weight: 500;
    text-align: center;
    color: #FFFFFF;
    margin-top: 5%;
    text-transform: uppercase;
    letter-spacing: 4px;
}

form.login
{
    width: 270px;
    margin: 0 auto;
}

form.login input[type="text"], form.login input[type="password"]
{
    width: 100%;
    margin: 0;
    padding: 10px 10px;
    background: 0;
    border: 0;
    border-bottom: 1px solid #FFFFFF;
    outline: 0;
    font-style: italic;
    font-size: 18px;
    font-weight: 600;
    letter-spacing: 1px;
    margin-bottom: 5px;
    color: #FFFFFF;
    outline: 0;
}

form.login input[type="submit"]
{
    width: 60%;
    font-size: 14px;
    text-transform: uppercase;
    font-weight: 500;
    margin-top: 36px;
    outline: 0;
    cursor: pointer;
    letter-spacing: 1px;
    display: block;
    margin : 0 auto;
    margin-top: 36px;
}

form.login input[type="submit"]:hover
{
    transition: background-color 0.5s ease;
}

form.login label, form.login a
{
    font-size: 12px;
    font-weight: 400;
    color: #FFFFFF;
}

form.login a
{
    transition: color 0.5s ease;
}

form.login a:hover
{
    color: #2ecc71;
}

.logoCol {
    height: 100%;
}

#logodiv svg#logo {
    width: 240px;
    height: 100%;
}

/** Re-style web-kit browser autocomplete boxes (Fixes Chrome's ugly yellow background) **/
@-webkit-keyframes autofill {
    to {
        color: white;
        background: transparent;
    }
}

input:-webkit-autofill {
    -webkit-animation-name: autofill;
    -webkit-animation-fill-mode: both;
}

  ```
</details>

### What you expected to happen
Icon aligned with field.

This is nifty feature. I appreciate the work being done and the recent fixes were a HUGE improvement. Thanks!

**Comment by stripthis** [comment]
Thanks for the report, we'll have a look!

**Comment by cedricalfonsi** [comment]
Fixed with v3.5
